languages/ruby/exercises/concept/exceptions/simple_calculator_test.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
```

Comment:
I can't find a string literal that needs to be frozen, one that could be mutated.  This could be left out, or could be set to false (which is just as bad, as there simply isn't any that needs to be unfrozen, and already they aren't risk of being mutated.)

Instead, if we are concerned that a given string is mutated (as in the literal string given as an argument for `SimpleCalculator`, there should be a test for that specifically.

Reply:
👍 This was copy and paste from other file, I didn't think about it